modyn/evaluator/internal/metrics/abstract_evaluation_metric.py

```diff
@@ -0,0 +1,46 @@
+from abc import ABC, abstractmethod
+
+import torch
+
+
+class AbstractEvaluationMetric(ABC):
```

Comment:
just a note: here we can keep the Metric in the name. Accuracy then is an instance of a AbstractEvaluationMetric

Reply:
Okay, I agree.